site/installation.md

```diff
@@ -11,7 +11,46 @@ Note that there is a short video demonstration of the installation process avail
 [comment]: # ( Note that you *must* create the `docker-registry` secret in the `weblogic-operator` namespace, so you will need to create the namespace first. )
 [comment]: # ( In this command, replace the uppercase items with the appropriate values. The `SECRET_NAME` will be needed in later parameter files.  The `NAMESPACE` must match the namespace where the operator will be deployed. )
 
-## Build the Docker image for the operator
+## Build the Docker image for the operator using Wercker
```

Comment:
Ryan not sure if the "Build the Docker image for the operator using Wercker" should not go immediately after "Build the Docker image for the operator locally" since this section includes requirements for the Operator image.

Reply:
@mriccell But, you don't need to meet any of these requirements to build using Wercker, since the customer will get a box from Wercker's quota and the wrecker.yml ensures that the requirements (java, maven) are satisfied.